manifests/java.yml

```diff
@@ -410,7 +410,54 @@ tests/:
             vertx3: missing_feature
             vertx4: missing_feature
     rasp/:
-      test_lfi.py: missing_feature
+      test_lfi.py:
+        Test_Lfi_BodyJson:
+          '*': v1.40.0
+          spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
+          spring-boot-payara: bug (produces 500 errors)
```

Comment:
Could you set a jira ticket here, to prevent loosing the info ? `spring-boot-payara: bug (APPSEC-0000)`

Same for later `bug` declarations

Reply:
I've also add it to other rasp test as is the same problem